celery/apps/beat.py

```diff
@@ -71,22 +77,32 @@ def __init__(self, max_interval=None, app=None,
         if not isinstance(self.loglevel, numbers.Integral):
             self.loglevel = LOG_LEVELS[self.loglevel.upper()]
 
-    def run(self):
+    @property
+    def app(self) -> Celery:
+        if self._app:
+            return self._app
+        raise ValueError("Beat must provided a Celery app.")
```

Comment:
> Added lines #L83 - L84 were not covered by tests

Can you add a unit test for this please?

Reply:
Will do for both!

Reply:
Fixed a type hint (`scheduler`/`scheduler_cls`), and added a test that covers both.

Reply:
👍

Reply:
I seem to have some trouble running unit tests locally, pytest must be caching (or something else, not sure) as the new field on the `Beat` class isn't even picked up when inspecting with `pdb`. Do you have any ideas here?